pkg/debug/scripts/collect-info.sh

```diff
@@ -44,17 +44,26 @@ DIR="$TMP_DIR/$INFO_DIR"
 mkdir -p "$DIR"
 mkdir -p "$DIR/network"
 
-# Install GNU version of the 'ps' and the dmidecode tool
-echo "- install GNU tools"
-apk add procps dmidecode >/dev/null 2>&1
+# Check for internet access, timeouts at 3 seconds
+nc -z -w 3 google.com 443 2>/dev/null
```

Comment:
@shjala , I did my comments in the old commit, I'm not sure if you saw it, so I will repost here (also for traceability):

You can use https://dl-cdn.alpinelinux.org/ instead of google.com since we only want internet access to install the alpine packages...

I think 3 seconds it's too short for a timeout, we just don't want to get the script stuck, so something like 30s would be reasonable I would say.... the common scenario is to have internet access...

Reply:
done.